app/controllers/my_modules_controller.rb

```diff
@@ -15,20 +15,14 @@ class MyModulesController < ApplicationController
   before_action :load_vars_nested, only: %I[new create]
```

Comment:
This and all the comments related task (and also completing task) was because of a typo: I should spell `can_manage_comment_in_module?` instead of `manage_comment_in_module?`.